hibernate-core/src/test/java/org/hibernate/jpa/test/ejb3configuration/PersisterClassProviderTest.java

```diff
@@ -302,12 +302,12 @@ public boolean hasLazyProperties() {
 		}
 
 		@Override
-		public Object load(Serializable id, Object optionalObject, LockMode lockMode, SharedSessionContractImplementor session) {
+		public Object load(Serializable id, Object optionalObject, LockMode lockMode, SharedSessionContractImplementor session, Boolean readOnly) {
```

Comment:
IMO this class should compile without changes

Reply:
Reverted back. Thanks.